src/platform/utilities/api/index.js

```diff
@@ -45,9 +45,16 @@ function isJson(response) {
  * @param {Function} **(DEPRECATED)** success - Callback to execute after successfully resolving
  * the initial fetch request.
  * @param {Function} **(DEPRECATED)** error - Callback to execute if the fetch fails to resolve.
+ * @param {string} apiVersion - 'v0' by default. Set to 'v1' if you need it.
  */
-export function apiRequest(resource, optionalSettings = {}, success, error) {
-  const baseUrl = `${environment.API_URL}/v0`;
+export function apiRequest(
+  resource,
+  optionalSettings = {},
+  success,
+  error,
+  apiVersion = 'v0',
```

Comment:
Oh! Actually could we put `apiVersion` within `optionalSettings` so as to reduce the amount of args for `apiRequest`? 🙂 I haven't looked at the code too much to see if that's feasible though